src/BlazorUI/Bit.BlazorUI/Components/ChoiceGroup/BitChoiceGroup.razor.cs

```diff
@@ -73,23 +75,21 @@ public bool IsRequired
     /// </summary>
     [Parameter] public EventCallback<BitChoiceGroupOption> OnChange { get; set; }
 
+
     protected override string RootElementClass => "bit-chg";
 
     protected override void RegisterComponentClasses()
     {
-        ClassBuilder.Register(() => IsEnabled && Options.Any(o => o.IsEnabled) && IsRequired
-                                   ? $"{RootElementClass}-required-{VisualClassRegistrar()}" : string.Empty);
+        ClassBuilder.Register(() => IsEnabled && Options.Any(o => o.IsEnabled) && IsRequired ? "required" : string.Empty);
 
-        ClassBuilder.Register(() => ValueInvalid is true
-                                   ? $"{RootElementClass}-invalid-{VisualClassRegistrar()}" : string.Empty);
+        ClassBuilder.Register(() => ValueInvalid is true ? "invalid" : string.Empty);
```

Comment:
I think better if this line moves to the base class. I do it in [this PR](https://github.com/bitfoundation/bitplatform/pull/3875).